pkg/apis/kops/dockerconfig.go

```diff
@@ -18,28 +18,34 @@ package kops
 
 // DockerConfig is the configuration for docker
 type DockerConfig struct {
-	Bridge   *string `json:"bridge,omitempty" flag:"bridge"`
-	LogLevel *string `json:"logLevel,omitempty" flag:"log-level"`
-	IPTables *bool   `json:"ipTables,omitempty" flag:"iptables"`
-	IPMasq   *bool   `json:"ipMasq,omitempty" flag:"ip-masq"`
-
-	LogDriver string   `json:"logDriver,omitempty" flag:"log-driver"`
-	LogOpt    []string `json:"logOpt,omitempty" flag:"log-opt,repeat"`
-
-	// Storage maps to the docker storage flag
-	// But nodeup will also process a comma-separate list, selecting the first supported option
-	Storage *string `json:"storage,omitempty" flag:"storage-driver"`
-
+	// AuthorizationPlugins is a list of authorization plugins
+	AuthorizationPlugins []string `json:"authorizationPlugins,omitempty" flag:"authorization-plugin,repeat"`
+	// Bridge is the network interface containers should bind onto
+	Bridge *string `json:"bridge,omitempty" flag:"bridge"`
+	// BridgeIP is a specific IP address and netmask for the docker0 bridge, using standard CIDR notation
+	BridgeIP *string `json:"bridgeIP,omitempty" flag:"bip"`
+	// DefaultUlimit is the ulimits for containers
+	DefaultUlimit []string `json:"defaultUlimit,omitempty" flag:"default-ulimit,repeat"`
+	// IPMasq enables ip masquerading for containers
+	IPMasq *bool `json:"ipMasq,omitempty" flag:"ip-masq"`
+	// IPtables enables addition of iptables rules
+	IPTables *bool `json:"ipTables,omitempty" flag:"iptables"`
+	// InsecureRegistry enable insecure registry communication @question according to dockers this a list??
```

Comment:
OK, that will need some careful handling, so we definitely don't want to do it in this PR :-)